source/src/main/groovy/com/kezong/fataar/FatLibraryPlugin.groovy

```diff
@@ -204,4 +206,24 @@ class FatLibraryPlugin implements Plugin<Project> {
             }
         }
     }
+
+    private void registerTransform() {
+        File manifest = project.android.sourceSets.main.manifest.srcFile
+        String packageName = new XmlParser().parse(manifest).@package
+        // 在 project.afterEvaluate 内部注册不生效
+        // 不在 project.afterEvaluate 内部获取不到扩展属性值
+        project.android.registerTransform(new RTransform(project, packageName))
+
+        project.afterEvaluate {
+            project.android.libraryVariants.all { variant ->
+                String variantName = variant.name.capitalize()
+                def transformTask = project.tasks.getByName("transformClassesWithRenameRFor${variantName}")
```

Comment:
As I wrote in [comment](https://github.com/kezong/fat-aar-android/issues/184#issuecomment-682585942), this task should depend on `mergeClasses` task